libmscore/edit.cpp

```diff
@@ -1053,9 +1053,26 @@ void Score::regroupNotesAndRests(const Fraction& startTick, const Fraction& endT
                                     continue; // this voice is empty here
                               if (!cr->isRest() || s->tick() + cr->actualTicks() > maxTick || toRest(cr)->isGap())
                                     break; // next element in the same voice is not a rest, or it exceeds the selection, or it is a gap
+                              if (cr->tuplet()) {
+                                    Tuplet* tuplet = cr->tuplet(); // element is a tuplet; special case
+                                    short tupletSize = tuplet->elements().size();
+                                    bool tupletAllRests = true;
+                                    for (auto const& el : tuplet->elements()) {
```

Comment:
FYI, we support nest tuplets too (tuplets within tuplets).  I haven't built this but am guessing this won't work correctly in those cases.  Assuming that's true, I wouldn't worry too much about it. but detecting it and handling it gracefully could be nice.  You'll know a nested tuplet because of the elements is itself another tuplet.

Reply:
For this reason, why not extend the Tuplet class with a method returning true when all element of the tuplet are rests? Using such method makes it also very easy to recursively call the method in case a nested tuplet is found.

Reply:
This is definitely a special case to consider, we will add a case to handle it gracefully, and create an issue to handle it properly. We will try to have this up tomorrow